src/game_display.cpp

```diff
@@ -717,9 +717,9 @@ std::vector<texture> game_display::get_reachmap_images(const map_location& loc)
 			std::string name;
 			stream << *image_prefix_;
 			if(tiles[i] == ENEMY) {
-				suffix = ".png~RC(magenta>red)";
+				suffix = ".png~RC(magenta>"+prefs::get().reach_map_enemy_color()+")~O("+std::to_string(prefs::get().reach_map_border_opacity())+"%)";
```

Comment:
I would declare these as local variables in `game_display::get_reachmap_images`, something along the lines of:

`string color = prefs::get().reach_map_color()`
`string enemy_color = prefs::get().reach_map_enemy_color()`
`string opacity = std::to_string(prefs::get().reach_map_border_opacity())`

So they can be used here (and Line 722) to improve code clarity. Also adding a spacebar between concatenations " + " may help.

Reply:
ok, implemented local variables